### ISSUE TITLE
can't install the plugin on raspberry-hosted grafana using CLI commands 

### ISSUE THREAD

**Issue by mo802** [geeks-r-us/mqtt-panel#39]
hello,
i'm trying to install the plugin on raspberry-hosted grafana using CLI commands but i get an error message.
the CLI command and the error message are shown bellow 
![plugin error](https://user-images.githubusercontent.com/67571553/132761456-f8cf8a1a-4689-4458-ae17-46b0484955ff.PNG)

**Comment by Garridus** [comment]
The sudo command allows users to gain administrative or root access
$ sudo grafana-cli

**Comment by mo802** [comment]
yeah really thanks, i think it successfully installed.
![plugin solved](https://user-images.githubusercontent.com/67571553/132766283-71b01c4c-141b-4d43-b81f-87c2550c60ad.PNG)

but where i can find this panel on grafana dashboard ?

**Comment by mo802** [comment]
i have found it but with a signature error !!
![plugin sign error](https://user-images.githubusercontent.com/67571553/132767270-11041c4d-4ee1-4289-b0f2-09d0be0ec4b0.PNG)

how could i solve this

**Comment by Garridus** [comment]
Edit grafana.ini
allow_loading_unsigned_plugins
Enter a comma-separated list of plugin identifiers to identify plugins to load even if they are unsigned. 

[plugins]
enable_alpha = true

;app_tls_skip_verify_insecure = false
# Enter a comma-separated list of plugin identifiers to identify plugins that are allowed to be loa>
allow_loading_unsigned_plugins = geekrus-mqtt-panel
# Enable or disable installing plugins directly from within Grafana.
plugin_admin_enabled = false
;plugin_admin_external_manage_enabled = false
;plugin_catalog_url = https://grafana.com/grafana/plugins/

**Comment by mo802** [comment]
sorry i'm totally beginner to this. i'd be so grateful if you clarify more how to do that!
i'm searching on how to edit grafana.ini but i found "INI customizations are not available under the Free version" !!!

**Comment by Garridus** [comment]
/etc/grafana/grafana.ini

**Comment by mo802** [comment]
ok i opened it 
![ini file grafana](https://user-images.githubusercontent.com/67571553/132769955-d78578e0-4a56-42cf-995f-7206e58d685e.PNG)

what should i do next ?

**Comment by Garridus** [comment]
uncommnet and add:
 allow_loading_unsigned_plugins = geekrus-mqtt-panel

**Comment by mo802** [comment]
i know that uncomment is to delete the ; char .. 
so, what is the line i should to uncomment ?!

**Comment by Garridus** [comment]
allow_loading_unsigned_plugins

**Comment by mo802** [comment]
aha ok , i made it . just save and restart grafana ? or there are any other configuration steps ??

**Comment by Garridus** [comment]
Only restart

**Comment by mo802** [comment]
i get the same error message !
![plugin sign error](https://user-images.githubusercontent.com/67571553/132772577-7ef83554-7287-47ca-9bf4-f02ff1d887f4.PNG)

**Comment by mo802** [comment]
that's my grafana.ini
![ini](https://user-images.githubusercontent.com/67571553/132773015-cf3d293f-9086-4021-877d-cbf5413f5c1a.PNG)

**Comment by Garridus** [comment]
geeksrus-mqtt-panel

**Comment by mo802** [comment]
right. the plugin is added correctly. i'm really so grateful for your help. the only last thing is how to connect it to my switch. i'm using a sonoff switch flashed with tasmota firmware. i should change the server ip and the subscribe and publish topic, right?
![grafana switch server](https://user-images.githubusercontent.com/67571553/132785427-d2ba8a55-16d7-4a67-a45f-f36255424931.PNG)

**Comment by FlorianMESSINA** [comment]
Hi all,

I'm using docker in order to test mqtt-panel plugin : 
docker run -d -p 3000:3000 -e "GF_INSTALL_PLUGINS=https://github.com/geeks-r-us/mqtt-panel/releases/download/v1.0.7/geeksrus-mqtt-panel-1.0.7.zip;mqtt-panel" --name=grafana grafana/grafana

I don't see the plugin in grafana, never ....

Someone could help me please ?

using : v8.5.0 grafana

Thanks
